src/Hosting.Services/HostBuilderExtensions.cs

```diff
@@ -101,6 +101,7 @@ public static IServiceCollection AddOmexServiceFabricDependencies<TContext>(this
 						options.ValidateOnBuild = true;
 						options.ValidateScopes = true;
 					})
+					.ConfigureLogging(builder => builder.AddOmexLogging())
```

Comment:
This registration suppose to happen couple lines above https://github.com/microsoft/Omex/blob/712c919c1c5f8370c5105279bbd6b67bd59b9401/src/Hosting.Services/HostBuilderExtensions.cs#L95C9-L95C41

Could we try to make `AddOmexServiceFabricDependencies` an extensions method on `HostBuilder`, and same for `AddOmexServices`, Ideally `AddOmexServices` should not allow be called on ServiceCollection (or at least marked as depricated) as I've mentioned on previous PR that introduced configuration.

Reply:
This last line basically just enables OmexLogger to read configuration, the logger itself is already added to services at that point. It would be correct to call it as early as possible, but that messes up execution context registration and breaks unit tests. There's no point trying to fix things now, we'll be adopting OTel and deleting old logging and activities code.